sdgx/data_models/inspectors/datetime.py

```diff
@@ -85,19 +97,30 @@ def detect_datetime_format(self, series: pd.Series):
         Returns:
                str: The datetime format that can parse all dates in the series, or None if no such format is found.
         """
+
+        def _is_series_fit_format(parsed_series, match_rate):
+            length = len(parsed_series)
+            cnt = 0
+            for i in parsed_series:
+                if i is False:
+                    cnt += 1
+            false_rate = cnt / length
```

Comment:
We cloud make it simple

```suggestion
            false_num = len((for i in parsed_series if i is False))
            false_rate = false_mum / length
```